README.md

```diff
@@ -134,6 +134,49 @@ becomes
       extra-lib-dirs:
         lib/darwin
 
+Multiple conditionals must fall under one `when`, for example:
+
+    when:
+      - condition: os(darwin)
+        extra-lib-dirs: lib/darwin
+      - condition: os(linux)
+        extra-lib-dirs: /usr/lib/x64
+
+becomes
+
+    if os(darwin)
+      extra-lib-dirs:
+        lib/darwin
+    if os(linux)
+      extra-lib-dirs:
+        /usr/lib/x64
+
+If conditionals are declared under different `when` clauses only the last one
+will make it to the `.cabal` file:
```

Comment:
I think it may not necessarily the last one.  This is again resulting from how YAML works.  This is an object (think of it as a hash map), duplicate keys are not allowed.